css/properties/overflow.json

```diff
@@ -96,6 +96,11 @@
               "samsunginternet_android": {
                 "version_added": false
               }
+            },
+            "status": {
+              "experimental": true,
```

Comment:
I marked the multiple-keyword syntax as experimental because it's only included in the CSS Overflow Module Level 3 Editor's Draft, as far as I can tell? Not sure if that's the correct way to determine experimental status :)